### DISCUSSION TITLE
Standard Deviation of Protein Abundances is Small

### DISCUSSION THREAD

**Discussion by DarioS** [vdemichev/DiaNN#139] [General]
I have a collaborator's dataset processed by DIA-NN version 1.7.12. We have noticed that the standard devation of all proteins is small. There are 1093 blood samples and 259 proteins I have received in a matrix. I understand missing value filtering has been done and some proteins were discarded if they were missing for more than 50% of samples per batch.

```r
dim(proteins)
  259 1093
summary(t(proteins)[, 1:6])
      A1BG          APOA4           APOB          APOC1          APOC2          ITIH1     
 Min.   :22.8   Min.   :21.6   Min.   :22.5   Min.   :22.4   Min.   :21.3   Min.   :22.0  
 1st Qu.:23.9   1st Qu.:24.0   1st Qu.:24.1   1st Qu.:23.6   1st Qu.:23.4   1st Qu.:22.9  
 Median :24.0   Median :24.3   Median :24.4   Median :23.9   Median :23.7   Median :23.0  
 Mean   :24.0   Mean   :24.2   Mean   :24.3   Mean   :23.9   Mean   :23.7   Mean   :23.0  
 3rd Qu.:24.1   3rd Qu.:24.5   3rd Qu.:24.6   3rd Qu.:24.1   3rd Qu.:24.0   3rd Qu.:23.1  
 Max.   :28.3   Max.   :28.0   Max.   :28.8   Max.   :28.5   Max.   :28.8   Max.   :29.2 
apply(proteins[1:6, ], 1, function(protein) quantile(protein, c(0.05, 0.95)))
      A1BG  APOA4   APOB  APOC1  APOC2  ITIH1
5%  23.610 23.514 23.636 23.249 22.933 22.606
95% 24.349 24.865 24.984 24.475 24.476 23.316
```

What is the typical standard deviation per protein to be expected and how should I troubleshoot this dataset further?

**Comment by vdemichev** [comment]
Hi Dario,

Yes, this kind of variation is how it is for log2-transformed protein quantities in healthy population.

Best,
Vadim
